### ISSUE TITLE
Debian install instructions for scala-cli are out of date

### ISSUE THREAD

**Issue by DavidGoodenough** [VirtuslabRnD/scala-cli-packages#17]
apt-key has been deprecated, instead keys should be put in /etc/apt/trusted.gpg.d, so the first line of the debian installation instructions should read:-

sudo curl -s --compressed -o /etc/apt/trusted.gpg.d/scala-cli.key "https://virtuslab.github.io/scala-cli-packages/KEY.gpg"

rather than:-

curl -s --compressed "https://virtuslab.github.io/scala-cli-packages/KEY.gpg" | sudo apt-key add -

**Comment by lwronski** [comment]
Hi @DavidGoodenough, Thanks for reporting. Indeed, this needs to be updated. I ran your modified command as well as the next one from the instructions, but encountered another error:

```
$  sudo curl -s --compressed -o /etc/apt/trusted.gpg.d/scala-cli.key "https://virtuslab.github.io/scala-cli-packages/KEY.gpg"
$ sudo curl -s --compressed -o /etc/apt/sources.list.d/scala_cli_packages.list "https://virtuslab.github.io/scala-cli-packages/debian/scala_cli_packages.list"
$ sudo apt update
Get:1 https://virtuslab.github.io/scala-cli-packages/debian ./ InRelease [1773 B]
Hit:2 http://deb.debian.org/debian bookworm InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Hit:4 http://deb.debian.org/debian-security bookworm-security InRelease
Err:1 https://virtuslab.github.io/scala-cli-packages/debian ./ InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1B92C29C942DFC9B
Reading package lists... Done
W: GPG error: https://virtuslab.github.io/scala-cli-packages/debian ./ InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1B92C29C942DFC9B
E: The repository 'https://virtuslab.github.io/scala-cli-packages/debian ./ InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by DavidGoodenough** [comment]
Apparently the line is more complicated that I had thought.  The first line should read:-

curl -sS "https://virtuslab.github.io/scala-cli-packages/KEY.gpg" | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/scala-cli.gpg

I got this from https://itsfoss.com/apt-key-deprecated/

It would appear that the key as downloaded is the wrong format and so the above fixes that.  Sorry for misleading.
